app/src/main/java/org/mozilla/fenix/components/PrivateShortcutCreateManager.kt

```diff
@@ -14,12 +14,30 @@ import org.mozilla.fenix.HomeActivity
 import org.mozilla.fenix.R
 import org.mozilla.fenix.home.intent.StartSearchIntentProcessor
 import java.util.UUID
+import android.content.pm.ShortcutManager
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
 
 /**
- * Handles the creation of pinned shortcuts.
+ * Handles the creation and existence of pinned shortcuts.
  */
 object PrivateShortcutCreateManager {
 
+    fun doesPrivateBrowsingPinnedShortcutExist(context: Context) : Boolean {
```

Comment:
The linter may complain about the extra space here.
```suggestion
    fun doesPrivateBrowsingPinnedShortcutExist(context: Context): Boolean {
```